front/src/app/components/forms/convention/VerificationActionButton.tsx

```diff
@@ -250,10 +264,12 @@ const inputLabelByStatus: Record<ConventionStatusWithJustification, string> = {
   DRAFT: "Précisez la raison et la modification nécessaire",
   REJECTED: "Pourquoi l'immersion est-elle refusée ?",
   CANCELLED: "Pourquoi souhaitez-vous annuler cette convention?",
+  DEPRECATED: "Pourquoi l'immersion est-elle obsolète",
```

Comment:
```suggestion
  DEPRECATED: "Pourquoi l'immersion est-elle obsolète ?",
```

Et tu peux aussi ajouter un espace avant le point d'intérogation sur la ligne `CANCELLED` ?